### PR TITLE
EDE-698 fix failing tests due to segment changes

### PR DESCRIPTION
**Pull request by kashif938**
#### Story Link
https://edlyio.atlassian.net/browse/EDE-698

#### PR Description
Due to changes in segment; tests were being failed. fixed those failed tests.

#### Type of change

Please select the options that are relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Change

#### Checklist before merging:

- [ ] Squased
- [x] Reviewd

### CODE REVIEW

**Comment by edtecheco** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Comment by danialmalik** [comment]
jenkins run python

**Comment by edtecheco** [comment]
Your PR has finished running tests. There were no failures.

File: lms/djangoapps/shoppingcart/models.py [hunk 1]
```diff
@@ -529,7 +529,7 @@ def _emit_order_event(self, event_name, orderitems):
                 'revenue': str(self.total_cost),
                 'currency': self.currency,
                 'products': [item.analytics_data() for item in orderitems]
-            }, send_to_track=True, user=user)
+            }, send_to_track=True, user=self.user)
```
Comment (danialmalik) [inline lms/djangoapps/shoppingcart/models.py:532]: Thanks for finding this out

**Pull request merged**